frontend/src/widgets/OverviewWidgetField.scss

```diff
@@ -0,0 +1,51 @@
+@media( min-width: 1024px ){
+    .smart-hub--dashboard-overview-widget-field {
+        margin: 0;
+    }
+}
+
+.smart-hub--dashboard-overview-widget-field-icon-background-lg {
+    border-radius: 50%;
+    height: 5em;
+    width: 5em;
+}
+
+.smart-hub--dashboard-overview-widget-field-icon-background-sm {
+    border-radius: 50%;
+    height: 3em;
+    width: 3em;
+}
+
+.smart-hub--overview-widget-field-font-size {
+    font-size: 15px;
+}
+
+.smart-hub--overview-widget-field-data-font-size {
+    font-size: 19px;
+}
+
+
+.smart-hub--dashboard-overview-widget-field .smart-hub--ellipsis {
+    max-width: 200px;
+}
+
+// Make sure tooltip label text wraps were we expect.
+.smart-hub--dashboard-overview-widget-field-label .usa-tooltip span {
+   width: 242px;
+   display: block;
+   word-wrap:break-word;
+   white-space: normal;
+}
+
+// Vertical align for the question mark and tool tip button.
+.smart-hub--dashboard-overview-widget-field-label .usa-tooltip {
+    vertical-align: middle;
+}
+
+// Horizontally align the question mark with the tool tip button.
+.smart-hub--dashboard-overview-widget-field-label .usa-tooltip svg {
+    margin-left: 0px;
+    // For accessibility must be 24px.
+    height: 24px;
+    width: 24px;
```

Comment:
Can we switch to em to match the other heights in the file?

Reply:
I switched to em.